docs/sources/tempo/traceql/trace-structure.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Trace structure and TraceQL
+menuTitle: Trace structure and TraceQL
+description: Learn about trace structure and TraceQL queries.
+weight: 200
+keywords:
+  - syntax
+  - TraceQL
+---
+
+# Trace structure and TraceQL
+
+The TraceQL language is designed to use the same syntax and semantics as PromQL and LogQL, where possible.
+The differences being inherent to the very nature of searching spans and traces.
+
+## Trace structure
+
+Traces are telemetry data structured as trees.
+Traces are made of spans (for example, a span tree); there is a root span that can have zero to multiple branches that are called child spans.
+Each child span can itself be a parent span of one or multiple child spans, and so on so forth.
+
+![Trace_and_spans_in_tree_structure](/media/docs/tempo/traceql/trace-tree-structures-and-spans.png)
+
+In the specific context of TraceQL, a span has the following associated fields:
+
+- **name**: the span name
+- **duration**: difference between the end time and start time of the span
+- **status**: enum: `{ok, error, unset}`. For details, refer to [OTel span status](https://opentelemetry.io/docs/concepts/signals/traces/#span-status) documentation.
+- **kind**: enum: `{server, client, producer, consumer, internal, unspecified}`. For more details, refer to [OTel span kind ](https://opentelemetry.io/docs/concepts/signals/traces/#span-kind) documentation.
+- Attributes
+
+The first four properties are *intrinsics*.
+They are fundamental to the span.
+
+*Attributes* are custom span metadata in the form of key-value pairs.
+There are two types of attributes: **span attributes** and **resource attributes**.
```

Comment:
This is referring to the scope of the query.  I think we have an additional scope recently as well.

Reply:
@zalegrala Is this additional attribute available in the 2.5 release or for the 2.6 release? Our 2.5 docs say two attributes. 

If it's the latter, we could change it here and then remove that update from the 2.5 backport.

Reply:
Our TraceQL doc for 2.6 says "TraceQL has four different attribute scopes: span attributes, resource attributes, event attributes, and link attributes."

Reply:
Yep, you're right, good point.

Reply:
Actually, I could create a doc issue to update the attribute list for this page and also to check through the rest of the TraceQL doc to make sure that it's all up to date. That might be the better solution so everything is consistent.

Reply:
i'm glad you mentioned the other attributes. That's important to make sure that things are updated throughout.

Reply:
I've added it to the Impove TraceQL doc epic. #353